frontend/src/App.js

```diff
@@ -8,8 +8,20 @@ import ProfileEdit from "./components/profile/ProfileEdit";
 import ReviewList from "./components/ReviewList";
 import SignIn from "./components/SignIn";
 import SignUp from "./components/SignUp";
+import { useCookies } from "react-cookie";
 
 function App() {
+  const [cookies, setCookie] = useCookies(["auth_token"]);
```

Comment:
this is not doing anything here so we can remove it. Should already be handled in the signin